lib/global_id/uri/gid.rb

```diff
@@ -65,6 +65,10 @@ def create(app, model, params = nil)
         build app: app, model_name: model.class.name, model_id: model.id, params: params
       end
 
+      def create_from(app, klass, id, params = nil)
```

Comment:
Let's kill this and extend `create` to have this signature: `def create(app, model_name, model_id, params = nil)`.